docs/bridge/docs/03-RFQ/index.md

```diff
@@ -0,0 +1,209 @@
+      <tr>
+        <td><strong>Prove</strong></td>
+        <td>The [Relayer] submits an optimistic proof of their completed relay to the `originChain`.</td>
+      </tr>
+      <tr>
+        <td><strong>Guard</strong></td>
+        <td>After a proof is asserted for each bridge, a brief optimistic dispute period is initiated.
+        <br/><br/>During this period, [Guard] entities will verify that the proof is valid and accurate.</td>
+      </tr>
+      <tr>
+        <td><strong>Claim</strong></td>
+        <td>After the dispute period has passed, the [Relayer] may claim the user's originally deposited funds.
+        <br/><br/>This reimburses the [Relayer] for the funds which they delivered in the earlier [relay] step.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Entities
+
+<b>Users</b> <span style={{color: 'darkgray'}}><i>(Permissionless Role)</i></span>
+    <blockquote>
+        Uses a bridge interface to submit their intent to the RFQ system for a quote, and then to the chain for fulfillment.
+    </blockquote><br/>
+
+<b>Quoters</b> <span style={{color: 'darkgray'}}><i>(Permissioned Role)</i></span>
+    <blockquote>
+        Posts Passive and/or Active Quotes via the [Quoter API] to be matched against user bridge inputs.
+        <div></div>
+        <span style={{color: 'darkgray', fontSize: '0.9em'}}><i>Note: In practice, each participating Quoter typically acts as their own Relayer and Prover.</i></span>
+
+    </blockquote><br />
+
+<b>Relayers</b> <span style={{color: 'darkgray'}}><i>(Permissionless Role)</i></span>
+    <blockquote>
+        Observes bridge deposits and submits [relay] transactions to fulfill them.
+        <div></div>
+        Also submits `claim` transactions to be reimbursed for their relays.
+        <div></div>
+        <span style={{color: 'darkgray', fontSize: '0.9em'}}><i>Note: In practice, each participating Relayer typically acts as their own Quoter and Prover.</i></span>
+    </blockquote><br />
+
+<b>Provers</b> <span style={{color: 'darkgray'}}><i>(Permissioned Role)</i></span>
+    <blockquote>
+        Observes [relay]s and submits [prove] transactions to initiate relayer reimbursements.
+       <div></div>
+        <span style={{color: 'darkgray', fontSize: '0.9em'}}><i>Note: In practice, each participating Prover typically acts as their own Quoter and Relayer.</i></span>
+    </blockquote><br />
+
+<b>Guards</b> <span style={{color: 'darkgray'}}><i>(Permissioned Role)</i></span>
+    <blockquote>
+        Validates proofs during the [Dispute Period] and submits [dispute] transactions for any discrepancies found.
+        <div></div>
+        Currently, Synapse itself is currently the sole Guard operator of the Synapse RFQ system.
+    </blockquote><br />
+
+<b>Cancelers</b> <span style={{color: 'darkgray'}}><i>(Permissioned Role)</i></span>
+    <blockquote>
+        Allows pre-emptive cancellation of bridge requests which have already been deposited but have not yet been relayed.
```

Comment:
I would specify that the Cancelers can only cancel the expired requests (wrt their deadline).

Reply:
done